<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>921</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>921</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №9</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Алгоритм плавающего горизонта&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>cos(x) * sin(z)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sqrt(fabs(x * z))</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>exp(cos(x) * sin(z))</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sin(x * z)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>320</height>
         </size>
        </property>
        <property name="title">
         <string>Параметры</string>
        </property>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>X start</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>30</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>X end</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTextEdit" name="x_start">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="x_end">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Z start</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>180</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Z end</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTextEdit" name="z_start">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="z_end">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="x_step">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>X step</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTextEdit" name="z_step">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Z step</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>250</height>
         </size>
        </property>
        <property name="title">
         <string>Поворот (в градусах)</string>
        </property>
        <widget class="QTextEdit" name="angle_x">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>60</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="angle_y">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>130</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>по оси X</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>100</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>по оси Y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTextEdit" name="angle_z">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>200</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>170</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>по оси Z</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="plotBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Отобразить</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Очистить плоскость</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Время выполнения</string>
         </property>
        </widget>
        <widget class="QLabel" name="timeLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>0 мс</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="text">
    <string>Загрузить данные</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
